config/services/b2.toml

```diff
@@ -1,14 +1,18 @@
-# rustic config file to use opendal storage - in this case b2
+# rustic config file to use B2 storage via Apache OpenDAL
 [repository]
 repository = "opendal:b2" # just specify the opendal service here
 password = "mypassword"
+# or 
+# password-file = "/home/<username>/etc/secure/restic_passwd"
 
-# all other options are given here.
-# Please refer to https://opendal.apache.org/docs/rust/opendal/services/index.html for the options supported by each service
+# B2 specific options
 [repository.options]
 # Here, we give the required b2 options, see https://opendal.apache.org/docs/rust/opendal/services/struct.B2.html
-root = "repo-path" # Set the work directory for backend
-key_id = "my_id" # B2 application key keyID
-application_key = ".my_key" # B2 application key applicationKey
-bucket = "bucket_name" # B2 bucket name
-bucket_id = "bucket_id" # B2 bucket_id
+# These are required either here or set by environment variables
+# - See: https://github.com/apache/opendal/blob/39e0a69c8eadc44391ba0420fdc7de2a42962ecf/.env.example#L173-L178
+# - Issue pending: https://github.com/rustic-rs/rustic/issues/1073
```

Comment:
Please remove it here. Once we takle #1073 we can add it. Current state is it isn't available through env variables.